doc/2/guides/getting-started/2-storing-and-accessing-data/index.md

```diff
@@ -0,0 +1,205 @@
+description: Create and retrieve documents
+order: 200
+---
+
+# Storing and Accessing Data
+
+Now that Kuzzle is running, we are going to create some documents.
+
+Kuzzle organizes the data storage in 4 levels: 
+  - indexes
+  - collections
+  - documents
+  - fields
+
+An index brings together several collections, which in turn contain several documents, each of which is composed of several fields.
+
+The collections have mappings that characterize the way Elasticsearch will index the data for the search.
+
+::: info
+It is recommended that you define your own mappings in order to specify the desired types for your collections and avoid default types that may be imprecise.
+:::
+
+If you're more familiar with the way relational databases store data, here is how it compares:
+
+| Relational databases storage | Document-oriented storage |
+| :--------------------------: | :-----------------------: |
+| database                     | index                     |
+| table                        | collection                |
+| schema                       | mappings                  |
+| line                         | document                  |
+| column                       | field                     |
+
+### Prepare the database
+
+First, we are gonna create an index with Kourou: `kourou index:create nyc-open-data`
+
+::: info
+This will use the [index:create](/core/2/api/controllers/index/create) API action.
+:::
+
+Then, we are gonna create a collection inside this index. We will provide the following basic mappings:
+
+```js
+{
+  properties: {
+    name: { type: "keyword" },
+    city: { type: "keyword" },
+    age: { type: "integer" }
+  }
+}
+```
+
+Run the following command to create our `yellow-taxi` collection: 
+
+```bash
+$ kourou collection:create nyc-open-data yellow-taxi '{
+  properties: {
+    name: { type: "keyword" },
+    city: { type: "keyword" },
+    age: { type: "integer" }
+  }
+}'
+```
+
+::: info
+This will use the [collection:create](/core/2/api/controllers/index/create) API action.
+:::
+
+
+### Create some documents
+
+Now we have a collection ready to receive documents, again use Kourou to create one:
+
+```bash
+$ kourou document:create nyc-open-data yellow-taxi '{
+  name: "Yagmur",
+  city: "Antalya",
+  age: 27
+}'
+```
+
+::: info
+This will use the [document:create](/core/2/api/controllers/index/create) API action.
+:::
+
+Finally, we are going to use the [Admin Console](http://console.kuzzle.io) to look at what we have created.
+
+Select the `nyc-open-data` index and then the `yellow-taxi` collection. You should see one document in this collection.
+
+![admin console show document](./admin-console-show-document.gif)
+
+### Search for documents
+
+Un des meilleur atouts de Kuzzle est d'exposer directement de façon sécurisé la fonction de recherche d'Elasticsearch.
+
+Nous allons pouvoir utiliser le puissant langage de requête d'Elasticsearch pour récupérer les documents qui nous intéressent. 
```

Comment:
Missing translation. And I'm not fond of qualifying that as "one of kuzzle's best asset" since all we do is to expose someone else's work.

```suggestion
Kuzzle directly exposes Elasticsearch's query language in a secure way. 

We'll now use that to search for the documents we're interested in.
```